packages/istanbul-lib-coverage/lib/file-coverage.js

```diff
@@ -320,7 +411,7 @@ class FileCoverage {
         ret.branches = this.computeBranchTotals('b');
         // Tracking additional information about branch truthiness
         // can be optionally enabled:
-        if (this['bt']) {
```

Comment:
huh, typo before?

Reply:
Seems that way!

Reply:
If this was intentional, then the code should have a test, because it wasn't being covered.

Reply:
digging through #639 and linked efforts, seems like this was simply a typo